upgrade/upgrade.go

```diff
@@ -231,6 +243,75 @@ func ConfigureDollarModule(ctx sdk.Context, dollarKeeper *dollarkeeper.Keeper) (
 	}
 }
 
+// ConfigureSwapModule creates an initial USDC/USDN swap pool.
+func ConfigureSwapModule(ctx sdk.Context, swapKeeper *swapkeeper.Keeper, accountKeeper authkeeper.AccountKeeper) (err error) {
```

Comment:
Nit: could we alphabetize these 🙃

```suggestion
func ConfigureSwapModule(ctx sdk.Context, accountKeeper authkeeper.AccountKeeper, swapKeeper *swapkeeper.Keeper) (err error) {
```